xbmc/guilib/StereoscopicsManager.cpp

```diff
@@ -509,6 +509,8 @@ void CStereoscopicsManager::OnPlaybackStarted(void)
   STEREOSCOPIC_PLAYBACK_MODE playbackMode = (STEREOSCOPIC_PLAYBACK_MODE) CSettings::GetInstance().GetInt(CSettings::SETTING_VIDEOPLAYER_STEREOSCOPICPLAYBACKMODE);
   RENDER_STEREO_MODE mode = GetStereoMode();
 
+  g_infoManager.UpdateAVInfo();
```

Comment:
or, alternatively to asking the videoplayer (which is probably a bad idea architecture wise), maybe fire off a new signal that either passes the fileitem or it's stream details around? Question then still is how to prevent the double resolution change (first change when playback is started, second when 3D mode is enabled)